### PR TITLE
Fix/ad performance adapter

### PR DESCRIPTION
**Pull request by axelazaid**
* updated fields in `bing_ad_performance_adapter` model

### CODE REVIEW

File: models/adapters/bing_ad_performance_adapter.sql [hunk 1]
```diff
@@ -8,19 +8,24 @@ bing_keyword_performance_agg as (
...
         utm_term,
+        campaign_id,
+        campaign_name,
         'bing ads' as platform,
         sum(clicks) as clicks,
         sum(impressions) as impressions,
         sum(spend) as spend
     from bing_keyword_performance
-    group by 1, 2, 3, 4, 5, 6, 7, 8, 9
+    group by 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14
```
Comment (jthandy) [inline models/adapters/bing_ad_performance_adapter.sql]: can you switch this to `dbt_utils.group_by()`?

File: macros/stitch/base/stitch_bing_keyword_performance.sql [hunk 2]
```diff
@@ -13,6 +12,85 @@ with source as (
...
+        ad_group_name,
+        ad_id,
+        campaign_id,
+        campaign_name,
+        campaign_status,
+        
+        url,
+        split_part(split_part(split_part(url, '//', 2),'/', 1),'?',1)::varchar as url_host,
+        '/' || split_part(url, '/', 4)::varchar as url_path,
+        split_part(split_part(url,'utm_source=',2), '&', 1) as utm_source,
```
Comment (jthandy) [inline macros/stitch/base/stitch_bing_keyword_performance.sql]: switch these to be `dbt_utils.get_url_parameter()`

**Review by jthandy — CHANGES_REQUESTED** [review]
Really small, two things that should really be pulling from dbt_utils now that we have the package manager up and running.

File: models/adapters/bing_ad_performance_adapter.sql (hunk 1 above)
Comment (axelazaid) [reply models/adapters/bing_ad_performance_adapter.sql]: done

File: macros/stitch/base/stitch_bing_keyword_performance.sql (hunk 2 above)
Comment (axelazaid) [reply macros/stitch/base/stitch_bing_keyword_performance.sql]: done (long over-due, lol)

**Comment by jthandy** [comment]
@axelazaid let's wait another day on this one--i think i'll have `get_url_path()` knocked out by then.

File: packages.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+packages:
+ - package: fishtown-analytics/dbt_utils
```
Comment (drewbanin) [inline packages.yml:2]: :+1:

**Pull request merged**